pkg/apis/maistra/v2/extension_providers.go

```diff
@@ -19,9 +19,9 @@ type ExtensionProviderEnvoyExternalAuthorizationHTTPConfig struct {
 	// service defined by the Kubernetes service or ServiceEntry.
 	//
 	// Example: "my-ext-authz.foo.svc.cluster.local" or "bar/my-ext-authz.example.com".
-	Service string `json:"service"`
+	Service *string `json:"service"`
 	// REQUIRED. Specifies the port of the service.
-	Port int64 `json:"port"`
+	Port *int64 `json:"port"`
```

Comment:
If the port is required, why is this a pointer? By convention, pointers in an API are used to indicate that a value is optional.

Reply:
Right, I will revert this change.

Reply:
Done